.evergreen/run-tests.sh

```diff
@@ -121,5 +121,5 @@ if [ "$SLOW_TESTS_ONLY" == "true" ]; then
 else
     ./gradlew -PjdkHome=/opt/java/${JDK} -Dorg.mongodb.test.uri=${MONGODB_URI} \
               -Dorg.mongodb.test.awsAccessKeyId=${AWS_ACCESS_KEY_ID} -Dorg.mongodb.test.awsSecretAccessKey=${AWS_SECRET_ACCESS_KEY} \
-              ${TRANSACTION_URI} ${GRADLE_EXTRA_VARS} ${ASYNC_TYPE} --stacktrace --info test
+              ${TRANSACTION_URI} ${GRADLE_EXTRA_VARS} ${ASYNC_TYPE} --stacktrace --info --continue test
```

Comment:
This is to make evergreen builds continue even after one sub-project's tests fail, so we can see all the failed tests.

Reply:
ack